data/app-generator.appdata.xml.in

```diff
@@ -12,17 +12,21 @@
 	</description>
 	<screenshots>
 		<screenshot type="default">
-			<image>https://raw.githubusercontent.com/elementary-community/app-generator/refs/heads/main/data/io.github.ecommunity.app-generator.png</image>
+			<image>https://raw.githubusercontent.com/elementary-community/app-generator/refs/heads/main/data/io.github.ellie_commons.app-generator.png</image>
```

Comment:
Do we rename the URL in another PR? (it would be automatically redirected thanks to GitHub though)